ishell.go

```diff
@@ -495,31 +496,44 @@ func (s *Shell) multiChoice(options []string, text string, init []int, multiResu
 				selected = toggle(selected, cur)
 			}
 		}
-		update()
+		select {
```

Comment:
theoretically you can press arrows faster than `update()` function is executed... nevermind, I will remove `select`.